src/libraries/Microsoft.PowerFx.Core/Localization/Strings.cs

```diff
@@ -449,6 +449,19 @@ internal static class TexlStrings
         public static StringGetter IndexArg1 = (b) => StringResources.Get("IndexArg1", b);
         public static StringGetter IndexArg2 = (b) => StringResources.Get("IndexArg2", b);
 
+        public static StringGetter AboutDEC2HEX = (b) => StringResources.Get("AboutDEC2HEX", b);
```

Comment:
>AboutDEC2HEX [](http://example.com/codeflow?start=78&length=12)

Those values come from the resources - you need to add them at strings\PowerFxResources.en-US.resx (don't worry about the other languages; the localization team will translate them)